doc/playerctl.1.in

```diff
@@ -1,123 +1,265 @@
+.It Cm previous
+Command the player to skip to the previous track.
+.It Cm position Op Ar OFFSET Ns Op Ar + Ns | Ns Ar -
+Print the position of the current track in seconds.
+With
+.Ar OFFSET
+specified, seek to
+.Ar OFFSET
+seconds from the start of the current track.
+With the optional
+.Op Ar + Ns | Ns Ar -
+appended,
+seek forward or backward
+.Ar OFFSET
+seconds from the current position.
+.It Cm volume Op Ar LEVEL Ns Op + Ns | Ns Ar -
+Print the player's volume scaled from 0.0
+.Pq 0%
+to 1.0
+.Pq 100% .
+With
+.Ar LEVEL
+specified,
+set the player's volume to
+.Ar LEVEL .
+With the optional
+.Op Ar + Ns | Ns Ar -
+appended,
+increase or decrease the player's volume by
+.Ar LEVEL .
+.It Cm metadata Op Ar KEY
+Print available metadata information for the current track.
+When
+.Ar KEY is
+specified,
+print the value of
+.Ar KEY .
+.It Cm open Ar URI
+Open
+.Ar URI
+in the player.
+.Ar URI
+may be the name of a file or an external URL.
+.It Cm shuffle Op Ic On | Off
+Print the shuffle status of the player.
+With the shuffle status specified,
+set shuffle to either
+.Ic On
+or
+.Ic Off .
+.It Cm loop Op Ic None | Track | Playlist
+Print the loop status of the player.
+With the loop status specified,
+set the loop status to
+.Ic None
+.Pq disable looping ,
+.Ic Track
+.Pq loop the current track ,
+or
+.Ic Playlist
+.Pq loop the current playlist .
+.El
+.Ss Format Strings
+The output of the
+.Cm position ,
+.Cm metadata ,
+.Cm status
+and
+.Cm volume
+commands can be controlled using a format string.
+Variables set by these commands can be included in the format string
+by enclosing them in curly braces,
+i.e:
+.Brq Brq Va var .
+These will then be expanded on output.
+.Pp
+Each command has access to the following variables:
+.Bl -tag -width Ds
+.It Va playerName
+The name of the current player.
+.It Va position
+The time position of the current track,
+in microseconds.
+.It Va status
+The status of the current player.
+.It Va volume
+The player's volume scaled from 0.0
+.Pq 0%
+to 1.0
+.Pq 100% .
+.El
+.Pp
+Variables set by
+.Cm metadata
+are as follows:
+.Bl -tag -width Ds
+.It Va album
+The album of the current track.
+.It Va artist
+The artist of the current track.
+.It Va xesam:contentCreated
+The release date of the current track.
+.It Va mpris:length
+The total length of the current track,
+in microseconds.
+.It Va title
+The title of the current track.
+.It Va mpris:trackid
+The ID of the current track.
+.El
+.Pp
+Helper functions are also available to transform expanded variables into
+other representations.
+They are called in the form
+.Brq Brq Fn func var .
+The helper functions are as follows:
+.Bl -tag -width Ds
+.It Fn lc str
+Convert string
+.Va str
+to lowercase.
+.It Fn uc str
+Convert string
+.Va str
+to uppercase.
+.It Fn duration time
+Reformat the microsecond timestamp
+.Va time
+in
+.Ql hh:mm:ss
+form.
+Can only be called with
+.Va position
+or
+.Va mpris:length .
+.It Fn emoji var
```

Comment:
I want to leave `emoji()` undocumented because I'm planning on changing the signature and renaming it and I don't want that to be a breaking change for people.